src/components/App/index.js

```diff
@@ -30,12 +58,12 @@ export class App extends Component {
            <Switch>
              <Route exact path='/' component={StartScreen} />
              <Route exact path='/rules' component={RuleList}/>
-             <Route exact path='/new' component={NewGame} />
+             <Route exact path='/new' render={()=> <NewGame handleUserInit={this.handleUserInit}/>} />
```

Comment:
Question for future reference: Would this have been easier to implement using Redux?

Reply:
It may have been yes